src/bin/nydus-image/main.rs

```diff
@@ -671,17 +671,41 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
                         .required(true),
                 )
                 .arg(
-                    Arg::new("config")
-                        .long("config")
-                        .short('C')
-                        .help("config to compactor")
-                        .required(true),
+                    Arg::new("backend-type")
+                        .long("backend-type")
+                        .help("Type of backend [possible values: localdisk, s3, oss, registry, http-proxy]")
```

Comment:
Maybe provide a list method of backend types in `storage/src/factory.rs` and use it here.

Reply:
Yes, I will try it.